db/fixtures/miq_product_features.yml

```diff
@@ -4285,6 +4285,41 @@
             :description: Provision Configured Systems
             :feature_type: control
             :identifier: configured_system_provision
+  - :name: Ansible Job Templates
```

Comment:
This should probably be "Ansible Tower Job Template" if we're using the provider name.